.pipelines/templates/e2e-test-kind.yaml

```diff
@@ -1,54 +1,59 @@
 parameters:
   - name: driverWriteSecrets
     type: object
+  - name: helmTests
+    type: object
 
 jobs:
   - ${{ each driverWriteSecret in parameters.driverWriteSecrets }}:
```

Comment:
`--driver-write-secrets` is only applicable with helm. For the deploy manifests the default is `true` and we aren't overriding it later. So maybe we can just have the yaml tests as a separate job here instead of adding to the matrix?

Reply:
done